advanced_html_css/accessibility/accessibility_auditing.md

```diff
@@ -1,36 +1,36 @@
 ### Introduction
 
-By now you should feel confident enough to start making your websites a little more accessible for a lot of users. What can really help you make sure you're implementing certain a11y features correctly, though, is learning how to view the accessibility tree in your DevTools and how to audit your web pages for any outstanding a11y issues.
+By now you should feel confident enough to start making your websites a little more accessible for a lot of users. In this lesson, you will learn how to check if you've correctly implemented your a11y features with the help of DevTools.
 
 ### Learning outcomes
 By the end of this lesson, you should be able to:
 
 * Open the accessibility section within your browser's DevTools.
-* Audit a web page with a third party auditing tool.
+* Audit a web page with a third-party auditing tool.
 
 ### Accessibility DevTools
 
 Using your browser's DevTools is beyond useful for several things, from checking the styles applied to a page to debugging code, but you already know that! Here's something you may not know: you can even use the DevTools to look at various accessibility features as well, which can be great as a sort of "quick audit". You can check contrast ratios (as we mentioned in a previous lesson), view various accessibility properties, and view the accessibility tree, to name a few features.
 
 ### Accessibility auditing
 
-There are plenty of third party tools to audit the accessibility of a web page, each with their own pros and cons, though we're only going to mention three of those tools here. By getting into the habit of auditing your web pages, you'll be able to track down any outstanding a11y issues that you may have missed. If you decide to utilize one of these tools, or another auditing tool if you prefer one you come across, you should focus on fixing issues related to the concepts introduced in these lessons only for now.
+There are plenty of third-party tools to audit the accessibility of a web page, each with its own pros and cons. Here, we're only going to mention three of those tools. By getting into the habit of auditing your web pages, you'll be able to track down any outstanding a11y issues that you may have missed. Whether you decide to utilize one of these tools or any other auditing tool you prefer, you should only focus on fixing issues related to the concepts introduced in this lesson for now. 
```

Comment:
Just a nit for the last change:

```suggestion
There are plenty of third-party tools to audit the accessibility of a web page, each with its own pros and cons. Here, we're only going to mention three of those tools. By getting into the habit of auditing your web pages, you'll be able to track down any outstanding a11y issues that you may have missed. Whether you decide to utilize one of these tools or any other auditing tool you prefer, you should only focus on fixing issues related to the concepts introduced in this portion of the curriculum for now. 
```

I believe the original intention when this was written was to focus on the a11y concepts introduced overall in this part of the curriculum. So we should keep verbiage that expresses that.

Reply:
Agreed, I committed your suggestion.